dna-frontend/src/styles.scss

```diff
@@ -29,25 +29,7 @@
   --tui-base-02: #{c.$secondary-background};
   --tui-base-03: #{c.$tertiary-background};
   --tui-link: #{c.$primary-text};
-  --tui-link-hover: #{c.$secondary-text};
-
-  /* Size overrides */
-  --tui-height-xs: 2rem;
-  --tui-height-s: 2.75rem;
-  --tui-height-m: 3.5rem;
-  --tui-height-l: 5rem;
-  --tui-padding-xs: 0.5rem;
-  --tui-padding-s: 1rem;
-  --tui-padding-m: 2rem;
-  --tui-padding-l: 3rem;
-  --tui-radius-xs: 0.25rem;
-  --tui-radius-s: 0.5rem;
-  --tui-radius-m: 0.75rem;
-  --tui-radius-l: 1rem;
-  --margin-xs: 0.5rem;
-  --margin-s: 1rem;
-  --margin-m: 2rem;
-  --margin-l: 3rem;
```

Comment:
These classes were used by the login page
![image](https://github.com/Vero-Ventures/insurance-calculator/assets/97374451/2e943cc6-3d67-46ef-85de-2074d0f96051)

Reply:
Signup as well
![image](https://github.com/Vero-Ventures/insurance-calculator/assets/97374451/107c5a75-c20f-43ca-9832-666b35d767b1)

Reply:
rip